js/src/jsonrpc/generator/build-markdown.js

```diff
@@ -16,25 +16,46 @@
 
 import fs from 'fs';
 import path from 'path';
+import chalk from 'chalk';
 
+import { DUMMY } from '../helpers';
+import { BlockNumber } from '../types';
 import interfaces from '../';
 
-const MARKDOWN = path.join(__dirname, '../../release/interfaces.md');
+const ROOT_DIR = path.join(__dirname, '../docs');
 
-let preamble = '# interfaces\n';
-let markdown = '';
+if (!fs.existsSync(ROOT_DIR)) {
+  fs.mkdirSync(ROOT_DIR);
+}
+
+// Logging helpers
+function info (log) { console.log(chalk.blue(`INFO:\t${log}`)); }
+function warn (log) { console.warn(chalk.yellow(`WARN:\t${log}`)); }
+function error (log) { console.error(chalk.red(`ERROR:\t${log}`)); }
+
+const type2print = new WeakMap();
+
+type2print.set(BlockNumber, 'Quantity|Tag');
+
+function printType (type) {
+  return type2print.get(type) || type.name;
+}
 
 function formatDescription (obj, prefix = '', indent = '') {
   const optional = obj.optional ? '(optional) ' : '';
-  const defaults = obj.default ? `(default: ${obj.default}) ` : '';
+  const defaults = obj.default ? `(default: \`${obj.default}\`) ` : '';
```

Comment:
are we ok using " ` " character as a quote delimiter? More of it all over the code.